includes/Modules/Site_Verification.php

```diff
@@ -336,4 +385,103 @@ protected function setup_services( Google_Client $client ) {
 			'siteverification' => new Google_Service_SiteVerification( $client ),
 		);
 	}
+
+	/**
+	 * Handles receiving a verification token for a user by the authentication proxy.
+	 *
+	 * @since n.e.x.t
+	 */
+	private function handle_verification_token() {
+		$authentication = $this->authentication;
+		$auth_client    = $authentication->get_oauth_client();
+
+		if ( ! $auth_client->using_proxy()
+			|| empty( $_GET['googlesitekit_verification_token'] )
+			|| empty( $_GET['googlesitekit_verification_nonce'] )
```

Comment:
Let's continue to use `filter_input()` for accessing any request parameters which is recommended over direct access to request globals.

Reply:
The main reason I changed these to use the superglobal instead is to allow for setting/changing during tests. This isn't possible with `filter_input` as it only pulls from the initial request.
I don't currently have a test using these though, (perhaps one is needed?) so let me know what you'd like to do.

Reply:
Let's use `filter_input()` for now, we shouldn't compromise on code best practices for testability. In the long term, we can introduce a utility class for accessing such values where the base implementation use `filter_input()`, but it could be replaced with another implementation for unit tests that has direct access to the globals.

Reply:
https://pointybrackets.wordpress.com/2017/01/20/testing-values-filtered-with-filter_input-with-phpunit/

Reply:
> I have encountered this one in the past, [this is a simple fix](https://pointybrackets.wordpress.com/2017/01/20/testing-values-filtered-with-filter_input-with-phpunit/) which allows to keep our code testable without compromising on best practices.

I'd say it's okay using that for now, but in the long term, let's put this logic in a central utility class to be able to access it from anywhere. Otherwise we'll soon be re-creating `my_filter_input()` methods in every class it's needed.

Reply:
> Otherwise we'll soon be re-creating my_filter_input() methods in every class it's needed

For sure, PHP `filter_input()` itself is a function and we should definitely not create the same method over and over in various class.

> I'd say it's okay using that for now, but in the long term, let's put this logic in a central utility class to be able to access it from anywhere.
@felixarntz do we have todo about this, how are we keep track of this? Either in the code base or open an issue is find, it needs to be logged somewhere though.

Reply:
Done :) https://github.com/google/site-kit-wp/issues/886